### PR TITLE
junos network ee jobs update

### PR DESCRIPTION
**Pull request by rohitthakur2590**
Signed-off-by: Rohit Thakur <rohitthakur2590@outlook.com>
 
Depends-On: https://github.com/ansible/network-ee/pull/103

### CODE REVIEW

File: zuul.d/junipernetworks-junos-jobs.yaml [hunk 1]
```diff
@@ -80,179 +80,52 @@
...
 
 - job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-python36
-    parent: ansible-test-network-integration-junos-vsrx-network_cli
-    nodeset: vsrx3-18.4R1-python36
-    vars:
-      ansible_test_python: 3.6
+    name: network-ee-integration-tests-junos
+    parent: network-ee-integration-tests
+    nodeset: vsrx3-18.4R1-python38
```
Comment (pabelanger) [inline zuul.d/junipernetworks-junos-jobs.yaml:91]: we likely need to bump this nodeset to use ubuntu-bionic-2vcpu label. if you look at nodesets.yaml for vyos node.

**Review by pabelanger — CHANGES_REQUESTED** [review]
We also need to update zuul.d/project-templates.yaml to reference the new jobs and remove the old ones.

**Comment by pabelanger** [comment]
recheck

File: zuul.d/nodesets.yaml [hunk 2]
```diff
@@ -789,26 +758,11 @@
...
-      - name: controller
-        nodes:
-          - fedora-32
-
 - nodeset:
     name: vsrx3-18.4R1-python38
     nodes:
       - name: ubuntu-bionic
-        label: ubuntu-bionic-1vcpu
+        label: ubuntu-bionic-2vcpu
```
Comment (pabelanger) [inline zuul.d/nodesets.yaml:765]: lets move this to centos-8-stream

File: zuul.d/junipernetworks-junos-jobs.yaml [hunk 3]
```diff
@@ -79,180 +79,59 @@
...
 
 - job:
-    name: ansible-test-network-integration-junos-vsrx-network_cli-libssh-python36
-    parent: ansible-test-network-integration-junos-vsrx-network_cli-python36
+    name: network-ee-integration-tests-junos-libssh
```
Comment (pabelanger) [inline zuul.d/junipernetworks-junos-jobs.yaml:100]: we should name these network-ee-integration-tests-junos-vsrx, given we have a 2nd nodeset for junos

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

File: zuul.d/junipernetworks-junos-jobs.yaml [hunk 4]
```diff
@@ -4,49 +4,17 @@
 - job:
     name: ansible-network-junos-appliance
     parent: ansible-network-appliance-base
-    pre-run: playbooks/ansible-network-junos-appliance/pre.yaml
+    pre-run:
+      - playbooks/ansible-network-junos-appliance/pre.yaml
+      - playbooks/network-ee-integration-tests/pre.yaml
     run: playbooks/ansible-network-junos-appliance/run.yaml
```
Comment (pabelanger) [inline zuul.d/junipernetworks-junos-jobs.yaml:10]: this run needs to be removed, please refer to vyos-vyos-jobs.yaml for example

Comment (pabelanger) [reply zuul.d/junipernetworks-junos-jobs.yaml:10]: actually, I am wrong... and tired :)

Comment (rohitthakur2590) [reply zuul.d/junipernetworks-junos-jobs.yaml:10]: yeah, I see it should remain. :)

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Comment by rohitthakur2590** [comment]
recheck

**Pull request closed without merging**